docs/connecting/authentication.md

```diff
@@ -0,0 +1,42 @@
+When a client connection is established Supavisor needs to verify the credentials of the connection.
+
+Credential verificiation is done either via `user` records or an `auth_query`.
+
+## Tenant User Record
+
+If no `auth_query` exists on the `tenant` record credentials will be looked up from a `user` and verified against the client connection string credentials.
```

Comment:
if Supavisor does not find the specified user/alias in the users table, it will attempt to find a user with `is_manager: true` set to true and retrieve creds using the auth query

Reply:
"If the `user` in the client connection is not found for a `tenant` it will use the `user` where `is_manager` is `true` and the `auth_query` on the `tenant` to return matching credentials from the tenant database."